src/courses/views.py

```diff
@@ -581,6 +583,21 @@ def ajax_progress_chart(request, user_id=0):
                 {'x': day + 1, 'y': xp}
             )
 
+        # SAT + SUN are masked by default from numpy.busday_offset. Dont have to worry about it not being excluded
+        # have to use datetime.now() because get_date will sometimes be ahead by 1 day when using get_date() + fraction_complete()
+        today = make_aware(datetime.datetime.now())
+        if today.weekday() in [5, 6]:  # SAT or SUN
+            # will always be total_xp <= user.profile.xp_cached / xp_to_date
```

Comment:
I'm confused by this comment.  Can you clarify what thing `will always be total_xp...` and why the rest of the comment is important?  The connection isn't obvious (reading this out of context...)

Reply:
changed to 

```
# total_xp <= xp_to_date(today) since the latest xp_data day is the last valid day (usually a friday)
# ie. if today is sunday: friday's total xp <= sunday's total xp 
```